V2RayVpnService.kt: fix OS marks VPN as metered

akiirui opened the pull request:
VPN apps targeting Build.VERSION_CODES.Q or above will be considered metered by default.

Ref:
https://developer.android.com/reference/android/net/VpnService.Builder#setMetered(boolean)
https://developer.android.com/about/versions/pie/android-9.0-changes-all#network-capabilities-vpn

akiirui committed:
V2RayVpnService.kt: fix OS marks VPN as metered

VPN apps targeting Build.VERSION_CODES.Q or above will be considered metered by default.

Ref:
https://developer.android.com/reference/android/net/VpnService.Builder#setMetered(boolean)
https://developer.android.com/about/versions/pie/android-9.0-changes-all#network-capabilities-vpn

yuhan6665 commented:
@akiirui thanks for your contribution! It is indeed a issue we need to fix.
Although I read about the document you shared. I think we should add
`VpnService.Builder.setMetered(false)`

The line you changed is the network we request for. Adding it may cause our VPN to lose connection using 4G (metered)

yuhan6665 commented on V2rayNG/app/src/main/kotlin/com/v2ray/ang/service/V2RayVpnService.kt:
```diff
@@ -549,4 +550,3 @@ class V2RayVpnService : VpnService() {
         }
     }
 }
-
```
This is trivial but I just want to mention I've seen projects having issue in Linux without line breaks at the end of files.
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

akiirui replied:
I think L552 is just an empty line... And I'm sure L551 have `\n`

akiirui commented:
> @akiirui thanks for your contribution! It is indeed a issue we need to fix.
> Although I read about the document you shared. I think we should add
> `VpnService.Builder.setMetered(false)`
> 
> The line you changed is the network we request for. Adding it may cause our VPN to lose connection using 4G (metered)

Yep, you're right. `NET_CAPABILITY_NOT_METERED` maybe let Mobile data marked as unmetered network, `VpnService.Builder.setMetered(false)` is better.

akiirui committed:
V2RayVpnService.kt: use setMetered

setMetered to false let VPN network to inherit its meteredness from its underlying networks.

Revert `addCapability(NetworkCapabilities.NET_CAPABILITY_NOT_METERED)`

yuhan6665 commented:
Look good to me.
You are right about the extra line break at the end of file. I was blind ;)
